src/components/CreateTeam/container.js

```diff
@@ -60,7 +57,10 @@ export default Wrapped =>
       createTeam(request)
         .then(() => {
           this.setState({
-            teamSubmitted: true,
+            initialFormValues: {
+              selectedClient: '-1',
+              teamName: '',
+            },
```

Comment:
This won't be needed if you keep the initial values out of state. Not sure it's needed anyway because we don't change initialFormValues state anywhere else.